includes/UID.php

```diff
@@ -35,7 +37,7 @@ public static function fromBin($value) {
 	}
 
 	public static function generate() {
-		$hex = \UIDGenerator::newTimestampedUID88(16);
+		$hex = MediaWikiServices::getInstance()->getGlobalIdGenerator()->newTimestampedUID128(16);
```

Comment:
This changes the number of bits from 88 to 128? Does the time extraction still work?